### PR TITLE
Add love.system.getPowerInfo()

### PR DESCRIPTION
**Pull request by RobLoach**

### CODE REVIEW

File: src/luaapi/system.c [hunk 1]
```diff
@@ -7,19 +7,27 @@
...
-static int l_system_get_os(lua_State* state) {
-  lua_pushstring (state, SDL_GetPlatform ());
+static int l_system_getOS(lua_State* state) {
+  lua_pushstring (state, system_getOS());
  return 1;
 }
 
+static int l_system_getPowerInfo(lua_State* state) {
+  system_PowerState power = system_getPowerInfo();
+  lua_pushinteger(state, power.state);
```
Comment (RobLoach) [inline src/luaapi/system.c]: Wasn't quite sure of the best way to manage the [PowerState constant enum](https://love2d.org/wiki/PowerState). Should we switch it to a const char* ?

Comment (Murii) [inline src/luaapi/system.c]: I think it would be easier for the developer to just receive a handled `const char*` for "state" and the rest to be int.

**Comment by Murii** [comment]
Thanks for the contribution !

**Comment by Murii** [comment]
We should mention that: /proc/acpi/battery does not exist on every linux (tested on 3 machines so far) so the output of this function will not be very useful everytime. It's a SDL bug tho.

**Pull request merged**